grpc/src/main/java/com/linecorp/armeria/common/grpc/GrpcMeterIdPrefixFunction.java

```diff
@@ -78,25 +94,38 @@ public MeterIdPrefix completeRequestPrefix(MeterRegistry registry, RequestLog lo
     private static void addGrpcStatus(ImmutableList.Builder<Tag> tagListBuilder, RequestLog log) {
         String status = log.responseHeaders().get(GrpcHeaderNames.GRPC_STATUS);
         if (status != null) {
-            tagListBuilder.add(Tag.of("grpc.status", status));
+            tagListBuilder.add(statusTag(status));
             return;
         }
 
         status = log.responseTrailers().get(GrpcHeaderNames.GRPC_STATUS);
         if (status != null) {
-            tagListBuilder.add(Tag.of("grpc.status", status));
+            tagListBuilder.add(statusTag(status));
             return;
         }
 
         final HttpHeaders trailers = GrpcWebTrailers.get(log.context());
         if (trailers != null) {
             status = trailers.get(GrpcHeaderNames.GRPC_STATUS);
             if (status != null) {
-                tagListBuilder.add(Tag.of("grpc.status", status));
+                tagListBuilder.add(statusTag(status));
                 return;
             }
         }
 
-        tagListBuilder.add(Tag.of("grpc.status", String.valueOf(Code.UNKNOWN_VALUE)));
+        tagListBuilder.add(UNKNOWN_TAG);
+    }
+
+    private static Tag statusTag(String status) {
+        if ("0".equals(status)) {
```

Comment:
Question: I'm wondering how much this will be faster than just using the tag in the `STATUS_TAGS`. Do you have something in mind? 😄

Reply:
As a rule of thumb, I thought 99% responses return `"0"`. It would be efficient for that workload. But we need a benchmark to prove it. Let me share a score.

Reply:
The workload was 1000 for `"0"` and 160 for non-zero code values. 
```
Benchmark                     Mode  Cnt       Score       Error  Units
MeterIdBenchmark.withCheck0  thrpt    5  260411.604 ± 13934.391  ops/s
MeterIdBenchmark.hashLookup  thrpt    5  130794.652 ±  5804.394  ops/s
```

Reply:
Oh, that's so fast. 😄 
Thanks a lot for the benchmark. 🙇 
Could you add a comment on why we treat `0` especially?